tests/integration/Administration/Snippet/SnippetFinderTest.php

```diff
@@ -167,7 +167,6 @@ private function getResultSnippetsByCase(string $folder, string $localeInFilenam
 
         $reflectionClass = new \ReflectionClass(SnippetFinder::class);
```

Comment:
would be super nice, if you could refactor all the cases in tests to use the ReflectionHelper :innocent: :v:

Reply:
Mh, is the `ReflectionHelper` maybe a bit of an superfluous abstraction? In particular when now the `->setAccessible(true)` does not need to be set anymore. So I would actually try to deprecate the `ReflectionHelper` (or at least `ReflectionHelper::getMethod` and `ReflectionHelper::getProperty`), as for example in the `SitemapExporterTest` this approach cannot be used: https://github.com/shopware/shopware/pull/12718/files#diff-78028a4f73df15325b43e057e15581d5801f9a23f29c79a22a9e7e3e03cb93ec